--- stripe/admin/stripe.php
+++ stripe/admin/stripe.php
@@ -215,7 +215,7 @@
 print '<tr class="oddeven"><td>';
 print '<span class="titlefield fieldrequired">'.$langs->trans("STRIPE_TEST_SECRET_KEY").'</span></td><td>';
-print '<input class="minwidth300" type="text" name="STRIPE_TEST_SECRET_KEY" value="'.$conf->global->STRIPE_TEST_SECRET_KEY.'" placeholder="'.$langs->trans("Example").': ********************************">';
+print '<input class="minwidth300" type="password" name="STRIPE_TEST_SECRET_KEY" value="'.$conf->global->STRIPE_TEST_SECRET_KEY.'" placeholder="'.$langs->trans("Example").': ********************************">';
 print '</td><td></td></tr>';

@@ -277,7 +277,7 @@
 print '<tr class="oddeven"><td>';
 print '<span class="fieldrequired">'.$langs->trans("STRIPE_LIVE_SECRET_KEY").'</span></td><td>';
-print '<input class="minwidth300" type="text" name="STRIPE_LIVE_SECRET_KEY" value="'.getDolGlobalString('STRIPE_LIVE_SECRET_KEY').'" placeholder="'.$langs->trans("Example").': ********************************">';
+print '<input class="minwidth300" type="password" name="STRIPE_LIVE_SECRET_KEY" value="'.getDolGlobalString('STRIPE_LIVE_SECRET_KEY').'" placeholder="'.$langs->trans("Example").': ********************************">';
 print '</td><td></td></tr>';
